opentracing-spring-cloud-starter/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>io.opentracing.contrib</groupId>
     <artifactId>opentracing-spring-cloud-parent</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
```

Comment:
you need probably to rebase off master.

Reply:
done!

Reply:
There is still a change here and it should not be.

Reply:
the same in other pom files